packages/pfsense/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.19.3"
```

```suggestion
- version: "1.20.0"
```
I would make this a minor change, since it involves new functionality